test/commands/scanner/run.test.ts

```diff
@@ -772,31 +726,12 @@ describe('scanner:run', function () {
 						// make sure there's the right number of them.
 						expect(violations.length).to.equal(84, 'Should be 84 violations detected in the file');
 					});
-
```

Comment:
Moved this last. There is an issue with the `--verbose` flag. Any tests in a file after a test that uses `--verbose` will run as if it were verbose. Need to create a repro and file an issue with oclif.